src/main/app/src/static/smart-documents-result.html

```diff
@@ -0,0 +1,357 @@
+        --warning-bg: #fbedd5;
+        --warning-fill: #ffa500;
+      }
+
+      @font-face {
+        font-family: Roboto;
+        font-style: normal;
+        font-weight: 400 500;
+        src:
+          url("/assets/fonts/Roboto/400.woff2") format("woff2"),
+          url("/assets/fonts/Roboto/500.woff2") format("woff2");
+      }
+
+      body {
+        font-family: "Roboto", sans-serif;
+        font-size: 16px;
+        font-weight: 400;
+        line-height: 28px;
+        color: var(--primary-color);
+      }
+
+      /* Material Design Layout Structure */
+      .mdl-layout {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        height: 100%;
+        background-color: var(--background-color);
+      }
+
+      .mdl-layout__header {
+        background-color: var(--header-bar-color);
+        color: var(--secondary-color);
+        display: flex;
+        align-items: center;
+        height: 64px;
+        position: relative;
+        z-index: 3;
+      }
+
+      .mdl-layout__header-row {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100%;
+        padding: 0 16px;
+        width: 100%;
+      }
+
+      .mdl-layout--fixed-header .mdl-layout__header {
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        z-index: 4;
+      }
+
+      .mdl-layout__content {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: calc(100vh);
+        overflow-y: auto;
+        flex-grow: 1;
+        position: relative;
+        z-index: 1;
+      }
+
+      .mdl-layout-title {
+        font-size: 20px;
+        font-weight: 500;
+        margin: 0;
+        padding: 0 16px !important;
+        color: inherit;
+      }
+
+      .wrapper {
+        min-width: 444px;
+      }
+
+      .wizard-result {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        gap: 10px;
+        font-size: 14px;
+        line-height: 20px;
+        padding: 14px 16px;
+        margin-bottom: 16px;
+      }
+
+      .wizard-result.success {
+        background-color: var(--success-bg);
+        fill: var(--success-fill);
+      }
+
+      .wizard-result.failed,
+      .wizard-result.unknown {
+        background-color: var(--error-bg);
+        fill: var(--error-fill);
+      }
+
+      .wizard-result.canceled {
+        background-color: var(--warning-bg);
+        fill: var(--warning-fill);
+      }
+
+      svg {
+        fill: inherit;
+      }
+
+      .wizard-result > svg {
+        display: none;
+      }
+
+      .wizard-result.success > svg.checkmark,
+      .wizard-result.failed > svg.exclamation,
+      .wizard-result.canceled > svg.exclamation,
+      .wizard-result.unknown > svg.exclamation {
+        display: block;
+      }
+
+      .sub-text {
+        text-align: center;
+      }
+    </style>
+
+    <script>
```

Comment:
Same here. Does it make sense to have this in separate JS file?

Reply:
Same answer 😉

Reply:
Only con I would argue to split is have smaller chunks of code